SPECS/kernel/kernel-uki-dracut.conf

```diff
@@ -31,8 +34,8 @@ drivers+=" virtio-blk virtio-scsi "      # qemu-kvm
 drivers+=" hv-storvsc "                  # hyperv
 drivers+=" xen-blkfront "                # xen
 
-# root encryption
-drivers+=" dm_crypt "
```

Comment:
does dm_crypt need to be removed here?

Reply:
hmmm, good catch. I removed this early in testing, before I found the Parted fix. Let me add it back and verify.

Reply:
Re-added and everything is working as expected. Fixed in "round 1" of addressing comments 👍